core/src/main/java/nl/nn/adapterframework/jdbc/MessageStoreListener.java

```diff
@@ -72,6 +72,8 @@ public void configure() throws ConfigurationException {
 				sessionKeysList.add((String)stringTokenizer.nextElement());
 			}
 		}
+		setPreSelectQuery("SELECT TOP 1 MESSAGEKEY FROM IBISSTORE "
+				+ "WHERE TYPE = '" + JdbcTransactionalStorage.TYPE_MESSAGESTORAGE + "' AND SLOTID = '" + slotId + "' ");
```

Comment:
MessageStoreListener kan ongewijzigd (origineel) blijven, peekQuery default is dezelfde als selectQuery